### ISSUE TITLE
Templates in documentation aren't resolved

### ISSUE THREAD

**Issue by SethosII** [easybuilders/easybuild#446]
While searching in the list of supported software I noticed that templates aren't reoslved there. For example the homepage of `GffCompare` is displayed as `https://github.com/gpertea/%(namelower)s` instead of `https://github.com/gpertea/gffcompare`. See http://easybuild.readthedocs.io/en/latest/version-specific/Supported_software.html or https://github.com/easybuilders/easybuild/blob/master/docs/version-specific/Supported_software.rst.

**Comment by ocaisa** [comment]
Hmm, might need our BDFL for this one, it's a bit too deep for me to go looking. He's back in a couple of weeks.

**Comment by boegel** [comment]
Confirmed (see below), I'll look into getting this fixed, thanks for reporting @SethosII!

```
eb --list-software=detailed --output-format=rst | grep '%('
*homepage*: https://github.com/MikkelSchubert/%(namelower)s
*homepage*: http://cran.r-project.org/web/packages/%(name)s
*homepage*: http://%(namelower)s.readthedocs.io/en/latest/index.html
*homepage*: http://cran.r-project.org/web/packages/%(name)s
*homepage*: https://%(namelower)s.readthedocs.org/
*homepage*: https://homes.cs.washington.edu/~dcjones/%(name)s/
*homepage*: http://www.bioinformatics.babraham.ac.uk/projects/%(namelower)s/
*homepage*: https://github.com/gpertea/%(namelower)s
*homepage*: https://github.com/gpertea/%(name)s
*homepage*: https://ccb.jhu.edu/software/%(namelower)s
*homepage*: http://www.bioinf.uni-freiburg.de/Software/%(name)s/
*homepage*: http://www.netlab.uky.edu/p/bioinfo/%(name)s2
*homepage*: http://huttenhower.sph.harvard.edu/%(namelower)s/
*homepage*: http://cibiv.github.io/%(name)s/
*homepage*: http://smithlabresearch.org/software/%(name)s
*homepage*: http://%(namelower)s.org
*homepage*: https://biocore.github.io/%(namelower)s
*homepage*: https://github.com/ablab/%(namelower)s
*homepage*: http://fenderglass.github.io/%(name)s/
*homepage*: http://cran.r-project.org/web/packages/%(name)s
*homepage*: http://archive.broadinstitute.org/cancer/cga/%(namelower)s
*homepage*: http://www.bioinf.uni-leipzig.de/~kristin/Software/%(name)s/
*homepage*: https://github.com/COMBINE-lab/%(namelower)s
*homepage*: http://bioinf.spbau.ru/en/%(namelower)s
*homepage*: https://sourceforge.net/p/%(namelower)s/wiki/Home/
*homepage*: http://cran.r-project.org/web/packages/%(name)s
```

**Comment by boegel** [comment]
Problem is fixed in https://github.com/easybuilders/easybuild-framework/pull/2591, I'll update the generated documentation page once it's merged.

**Comment by boegel** [comment]
Now fixed with #460 after re-generating the documentation on top of the fix from https://github.com/easybuilders/easybuild-framework/pull/2591.

Thanks again for reporting this @SethosII!
